ports/spix/vcpkg.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "spix",
+  "version-string": "0.2",
```

```suggestion
  "version": "0.2",
```
This should be [version](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version).